docs/governance/mrpowers-io-org.md

```diff
@@ -0,0 +1,40 @@
+# mrpowers-io GitHub organization
+
+The mrpowers-io GitHub organization is owned by [Matthew Powers](https://www.linkedin.com/in/matthew-powers-cfa/) and [Sem Sinchenko](https://www.linkedin.com/in/semyon-a-sinchenko/) ("org owners").
+
+The org owners are responsible for overseeing projects and reviewing project reports on a quarterly basis.
+
+## Overseeing projects
+
+The org owners are responsible for voting on new projects that will be added in the mrpowers-io organization.
+
+The org owners are also responsible for monitoring the projects at a high-level including code quality, documentaion, the README, and repo activity.
+
+mrpowers-io is generally for repos that are being actively developed and used.
+
+## Managing repo permissions
+
+The org owners will have to manage repo permissions from time-to-time.
+
+The org owners may need to add a new maintainer to a repo if other repo owners don't have permissions, for example.
+
+## Overseeing package manager permissions
+
+Repos in the mrpowers-io organization will often be bundled and distributed via a package manager like Maven or PyPI.
+
+Repo maintainers should also have access to make releases via package managers.
+
+This ensures the package can continue being published in the event of an extreme event like a hardware failure, losing keys, or getting hacked.
+
+## Quarterly project reports
+
+Repo maintainers should submit project reports on a quarterly basis showing key metrics like the number or new contributors, number of new stars, PyPI downloads, etc.
+
+The report should explain how the repo maintainers plan to grow the project in the upcoming quarter.
+
+The mrpowers-io org is generally for projects that are being actively maintained with maintainers that are ambitiously growing the user base.
+
```

Comment:
I think that we should add a section about projects retirement. For example, if one day `chispa` will be moved to core pyspark we should archive it.

Reply:
Yea, this is outlined on the mrpowers-io-repos page.